cmd/avalanche-network-runner/control/control.go

```diff
@@ -90,8 +99,27 @@ func startFunc(cmd *cobra.Command, args []string) error {
 	}
 	defer cli.Close()
 
+	configFileContents := ""
+	if nodeConfigFilePath != "" {
+		color.Outf("{{yellow}}WARNING: overriding node configs with custom provided config file{{/}} %+v\n", nodeConfigFilePath)
+		fileBytes, err := os.ReadFile(nodeConfigFilePath)
+		if err != nil {
+			return fmt.Errorf("failed to read provided config file: %s", err)
```

Comment:
usually we use %w for errors

Reply:
https://pkg.go.dev/fmt#Errorf

Sometimes my linter warns me of errors which do not implement `Unwrap`. So when not sure, I use `%s`.